### PR TITLE
receiver names should be same

### PR DESCRIPTION
**Pull request by silenceshell**
Just rename the receiver name.

### CODE REVIEW

File: plugin/pkg/admission/security/podsecuritypolicy/admission.go [hunk 1]
```diff
@@ -95,10 +95,10 @@ func newPlugin(strategyFactory psp.StrategyFactory, failOnNoPolicies bool) *PodS
 	}
 }
 
-func (a *PodSecurityPolicyPlugin) SetExternalKubeInformerFactory(f informers.SharedInformerFactory) {
+func (plugin *PodSecurityPolicyPlugin) SetExternalKubeInformerFactory(f informers.SharedInformerFactory) {
```
Comment (errordeveloper) [inline plugin/pkg/admission/security/podsecuritypolicy/admission.go]: I was under impression that we only use single character names, but I agree that matching the rest of the functions is a good idea.

**Review by errordeveloper — APPROVED** [review]
/lgtm

**Comment by silenceshell** [comment]
@errordeveloper Would you please review the code again? I have just resolved some merge conflicts.

**Review by errordeveloper — APPROVED** [review]
/lgtm

**Comment by silenceshell** [comment]
hi @deads2k @krmayankk 
Would you help to review this PR? It is a little one, won't cost too much time, thanks very much.

**Comment by silenceshell** [comment]
hi @deads2k @krmayankk

Any opinion about this PR?

**Comment by nikhita** [comment]
/kind cleanup
/release-note-none

@silenceshell PR needs rebase. Apologies for the delay in response on this. If you don't recieve a response in a few days, please ping on #sig-auth or #pr-reviews channels on the Kubernetes slack. :grimacing:

**Comment by silenceshell** [comment]
@nikhita 
Thanks for your reply. I found the codes of master have renamed the receiver, so this PR is needless and I will close it.

**Pull request closed without merging**